bundles/org.openhab.binding.androiddebugbridge/src/main/java/org/openhab/binding/androiddebugbridge/internal/AndroidDebugBridgeDevice.java

```diff
@@ -202,6 +202,10 @@ public void stopPackage(String packageName)
     public String getCurrentPackage() throws AndroidDebugBridgeDeviceException, InterruptedException,
             AndroidDebugBridgeDeviceReadException, TimeoutException, ExecutionException {
         var out = runAdbShell("dumpsys", "window", "windows", "|", "grep", "mFocusedApp");
+        if (getAndroidVersion().contains("11")) {
+            out = runAdbShell("dumpsys", "window", "displays", "|", "grep", "mFocusedApp");
+            logger.debug("detected android version: {} ", getAndroidVersion());
```

Comment:
```suggestion
            logger.debug("detected android version 11: {} ", getAndroidVersion());
```

Reply:
what ever you think is best. I thought that the result of "getAndroidVersion()" is populated inside of those two brackets is why I had it that way .

Reply:
I'll withdraw this comment. I assumed that `getAndroidVersion()` would return some kind of longer string, and you wanted to log that the "version 11" path was chosen, based on that string.